test/Unit/TriggerBinding/SqlTriggerListenerTests.cs

```diff
@@ -201,24 +201,55 @@ public void ScaleMonitorGetScaleStatus_CountNotIncreasingOrDecreasing_ReturnsNon
             Assert.Contains("Requesting no-scaling: Found the number of unprocessed changes for table: 'testTableName' to not require scaling.", logMessages);
         }
 
+        [Theory]
+        [InlineData("1")]
+        [InlineData("10")]
+        [InlineData("10000")]
+        public void ScaleMonitorGetScaleStatus_VerifyUserConfiguredTriggerConfig(string maxChangesPerWorker)
+        {
+            (IScaleMonitor<SqlTriggerMetrics> monitor, List<string> logMessages) = GetScaleMonitor(maxChangesPerWorker);
+            ScaleStatusContext context = GetScaleStatusContext(new int[] { 100, 150, 130, 120, 90 }, 10);
+
+            monitor.GetScaleStatus(context);
+
+            Assert.Contains(logMessages, message => message.Contains($"maximum changes per worker: {maxChangesPerWorker}"));
```

Comment:
We should check beyond what is being logged. Please do not edit the tests for now. I will try pushing a commit for tests by EOD.